### PR TITLE
Show selcted Rundown item

### PR DESCRIPTION
**Pull request by dzonidoo**
SDESK-7188

link: https://github.com/superdesk/superdesk-ui-framework/pull/824

### CODE REVIEW

**Comment by dzonidoo** [comment]
first merge ui-framework side

File: scripts/extensions/broadcasting/src/interfaces.ts [hunk 1]
```diff
@@ -61,6 +61,7 @@ export interface IRundownItemBase {
         }
     };
     subitems?: Array<ISubitem>;
+    _id?: string;
```
Comment (petrjasek) [inline scripts/extensions/broadcasting/src/interfaces.ts:64]: that's defined in the `IBaseRestApiResponse` so should be available on `IRundownItem` type

File: scripts/extensions/broadcasting/src/rundowns/rundown-view-edit.tsx [hunk 2]
```diff
@@ -549,6 +549,11 @@ export class RundownViewEditComponent extends React.PureComponent<IProps, IState
                                                             ),
                                                         });
                                                     }}
+                                                    itemId={
```
Comment (petrjasek) [inline scripts/extensions/broadcasting/src/rundowns/rundown-view-edit.tsx:552]: not sure about the name here, maybe `selectedItem` would be better

File: scripts/extensions/broadcasting/src/interfaces.ts (hunk 1 above)
Comment (dzonidoo) [reply scripts/extensions/broadcasting/src/interfaces.ts:64]: i know that... but i have error in TS if i dont add in `IRundownItem` too

File: scripts/extensions/broadcasting/src/rundowns/components/rundown-items.tsx [hunk 3]
```diff
@@ -30,6 +30,7 @@ interface IPropsReadOnly<T extends IRundownItem | IRundownItemBase> {
     getActions(item: T): JSX.Element;
     preview(item: T): void;
     edit(item: T): void;
+    itemId?: string | null;
```
Comment (petrjasek) [inline scripts/extensions/broadcasting/src/rundowns/components/rundown-items.tsx:33]: I would change the name here as well

**Review by petrjasek — APPROVED** [review]

**Pull request merged**